modules/integration/src/test/scala/zio/elasticsearch/HttpExecutorSpec.scala

```diff
@@ -917,6 +917,48 @@ object HttpExecutorSpec extends IntegrationSpec {
             Executor.execute(ElasticRequest.createIndex(firstSearchIndex)),
             Executor.execute(ElasticRequest.deleteIndex(firstSearchIndex)).orDie
           ),
+          test("search for a document using a match boolean prefix query") {
```

Comment:
Write just one it-test for this query. There is no need to check `minimumShouldMatch` only.